framework/mesh/mesh_continuum/mesh_continuum.cc

```diff
@@ -62,9 +62,9 @@ MeshContinuum::MakeMPILocalCommunicatorSet() const
   {
     int locI_num_connections = static_cast<int>(local_connections.size());
 
-    // If chi::mpi.location_id == locI then this call will
+    // If mpi.location_id == locI then this call will
     // act like a send instead of receive. Otherwise
-    // It receives the count.
+    // it receives the count.
     mpi_comm.broadcast(locI_num_connections, locI);
```

Comment:
I would maybe even remove this comment. This is what broadcast does, so no need to explain how MPI works.

Reply:
done